shellescape/shellescape.go

```diff
@@ -0,0 +1,34 @@
+/*
+Package shellescape provides the shellescape.Quote to escape arbitrary
+strings for a safe use as command line arguments in the most common
+POSIX shells.
+
+The original Python package which this work was inspired by can be found
+at https://pypi.python.org/pypi/shellescape.
+*/
+package shellescape // "import gopkg.in/alessio/shellescape.v1"
```

Comment:
@ahuigo Can we remove this line? 
On the line, godoc has different meanings to it.
```
// "import gopkg.in/alessio/shellescape.v1"
```
If we learned something from this ` gopkg.in/alessio/shellescape.v1` library, please capture that acknowledgment in the above godoc instead of here.
Otherwise remove it.